config.json

```diff
@@ -748,8 +748,8 @@
       {
         "groupId": "com.google.firebase",
         "artifactId": "firebase-appcheck-interop",
-        "version": "16.0.0-beta04",
-        "nugetVersion": "116.0.0.7-beta04",
+        "version": "16.0.0",
+        "nugetVersion": "116.0.0",
```

Comment:
This will need to be `116.0.0.7`.

Reply:
correct.
Again. Tuning the versions and forgot to revert patch number